content/en/docs/setup/release/notes.md

```diff
@@ -1168,23 +1168,7 @@ filename | sha512 hash
 
 ### Other (Cleanup or Flake)
 
-- **Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
-  
```

Comment:
Hi @annajung .
I just looked at the generated page.
Should this line be present (3X), 
```
**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
```

Reply:
looks like there are multiple `Other (Cleanup or Flake)` headers. 
This line is only present in line 585 and not in line 1171

Reply:
linking the original changelog that I copied the content from in case it's helpful https://raw.githubusercontent.com/kubernetes/kubernetes/master/CHANGELOG/CHANGELOG-1.20.md

Reply:
OK. Let's get these changes in now? There may be a few more minor updates?

Reply:
yes, that sounds good. 
I'm not sure if there will be minor updates in the future? If so, then yeah, we should update accordingly.